app/services/clients/connector.client.js

```diff
@@ -132,7 +69,7 @@ ConnectorClient.prototype = {
    *@return {Promise}
    */
   getAccounts: async function (params) {
-    const url = `${this.connectorUrl}${_accountsUrlFor(params.gatewayAccountIds)}`
+    const url = `${this.connectorUrl}/v1/api/accounts?accountIds=` + params.gatewayAccountIds.join(',')
```

Comment:
This `params.gatewayAccountIds.join(',')` should probably also be encoded (`encodeURIComponent(params.gatewayAccountIds.join(','))`), or if we want the comma character left unencoded it can be `params.gatewayAccountIds.map(encodeURIComponent).join(',')`.

Reply:
Oops, I missed this. Will fix this in a next PR.